src/main/java/com/google/sps/data/DatastoreClientInterface.java

```diff
@@ -0,0 +1,54 @@
+package com.google.sps.data;
+
+import java.util.Date;
+import com.google.appengine.api.datastore.Entity;
+import java.util.Optional;
+import java.util.List;
+
+/** Interface for the DatastoreClient class. */
+public interface DatastoreClientInterface {
+  /**
+   * Adds an AttendeeInterface object to the Datastore. If object already
+   * exist, the old one is replaced.  
+   */
```

Comment:
You're missing information. Check this out. This is the proper way to write a javadoc.

https://www.oracle.com/technical-resources/articles/java/javadoc-tool.html#format